skll/utilities/generate_predictions.py

```diff
@@ -26,7 +27,8 @@ class Predictor(object):
     predictions for feature strings.
     """
 
-    def __init__(self, model_path, threshold=None, positive_label=1, logger=None):
+    def __init__(self, model_path, threshold=None, positive_label=1,
+                 return_all_probabilities=False, logger=None):
```

Comment:
A better keyword argument name might be `all_labels`:
```suggestion
                 all_labels=False, logger=None):
```